templates/apiary-bucket-policy.json

```diff
@@ -94,10 +94,24 @@
             "Resource": "arn:aws:s3:::${bucket_name}/*",
             "Condition": {
                 "StringNotEquals": {
-                    "s3:x-amz-server-side-encryption": "AES256"
+                    "s3:x-amz-server-side-encryption": "${encryption}"
```

Comment:
Now that the user of this module can specify the encryption type in the schema array, should we have some terraform code that validates it is one of the only two valid values - `AES256` or `aws:kms`?

Reply:
Or at the very least, make that clear in the documentation.

Reply:
good point, do you think adding as comment to example usage in README is sufficient?

Reply:
I think if it is in both the README, and in that line in VARIABLES.md that I pointed out, that should be sufficient.  If a user doesn't read at least one of those, they are going to have lots of other issues too. :)